Jellyfin.Server/Models/JsonOptions.cs

```diff
@@ -0,0 +1,41 @@
+using System.Text.Json;
+
+namespace Jellyfin.Server.Models
+{
+    /// <summary>
+    /// Json Options.
+    /// </summary>
+    public static class JsonOptions
+    {
+        /// <summary>
+        /// Base Json Serializer Options.
+        /// </summary>
+        private static readonly JsonSerializerOptions _jsonOptions = new JsonSerializerOptions();
```

Comment:
You shouldn't share the same object to back the two properties, each property should be backed by it's own object

Reply:
I did this so we can easily change defaults without having to duplicate code

Reply:
Yeah, that makes sense but the way you've implemented it doesn't work. On this line here when you update the property  you're updating it for both profiles/properties since you've used the same backing object for both properties: https://github.com/jellyfin/jellyfin/pull/2932/files#diff-5db90b3f2c41ec58d0d96decdd357b08R23

I think what you want might be this so that you get a new object each time it is called.

```suggestion
        private static JsonSerializerOptions DefaultJsonOptions => new JsonSerializerOptions();
```